opm/simulators/wells/WellInterfaceGeneric.cpp

```diff
@@ -190,6 +190,68 @@ double WellInterfaceGeneric::rsRvInj() const
     return well_ecl_.getInjectionProperties().rsRvInj;
 }
 
+void WellInterfaceGeneric::initInjMult(const std::vector<double>& max_inj_mult)
+{
+    // prev_inj_multiplier_ will stay unchanged during the time step
+    // while inj_multiplier_ might be updated during the time step
+    this->prev_inj_multiplier_ = max_inj_mult;
+    // initializing the inj_multipler_ to be 1.0
+    this->inj_multiplier_ = std::vector<double>(max_inj_mult.size(), 1.);
+}
+
+void WellInterfaceGeneric::updateInjMult(std::vector<double>& inj_multipliers, DeferredLogger& deferred_logger) const
+{
+    if (inj_multipliers.size() != this->inj_multiplier_.size()) {
+        OPM_DEFLOG_THROW(std::runtime_error,
+                         fmt::format("We do not support changing connection numbers during simulation with WINJMULT "
+                                     "for well {}", name()),
+                         deferred_logger);
+    }
+
+    inj_multipliers = this->inj_multiplier_;
+}
+
+
+
+double WellInterfaceGeneric::getInjMult(const int perf,
+                                        const double bhp,
+                                        const double perf_pres) const
+{
+    assert(!this->isProducer());
+
+    double multipler = 1.;
```

Comment:
Do you want to write this as "multiplier" instead?  Right now the word is missing an "i" between "l" and "e".